src/gt4py/config.py

```diff
@@ -23,7 +23,7 @@
 
 # Default paths (taken from user's environment vars when possible)
 BOOST_ROOT: str = os.environ.get(
-    "BOOST_ROOT", os.environ.get("BOOST_HOME", os.path.abspath("/usr/local/include"))
+    "BOOST_ROOT", os.environ.get("BOOST_HOME", os.path.abspath("/usr/local"))
```

Comment:
Is this how it is installed in Ubuntu? Could always be manually specified with `BOOST_HOME`.

Reply:
This change is because when the code executed, the path actually resolved to `/usr/local/include/include`. And yes, `BOOST_HOME` can be specified as an environmental variable.

Reply:
Good fix then!

Reply:
Thanks!